constants/misc_constants.asm

```diff
@@ -16,3 +16,6 @@ DEF TEXT_DELAY_SLOW   EQU %101 ; 5
 	const_def 6
 	const BIT_BATTLE_SHIFT     ; 6
 	const BIT_BATTLE_ANIMATION ; 7
+
+; D732 flags
+DEF BIT_DEBUG_MODE EQU 1
```

Comment:
pokecrystal tends to use something like `DEBUG_MODE_F` for bit flags; which is what this appears to be. @Rangi42 @dannye In the future should we consider standardizing this across gen1 as well?

Reply:
Yes, although standardizing the bitflag constant name conventions would be a separate issue+PR.